webroot/js/main.js

```diff
@@ -8,15 +8,25 @@ function doApiCall(method, url, outputDiv, errorDiv, body) {
         type: method,
         url: url,
         dataType: 'text',
-        headers: {
-            "Authorization": authHeader
-        },
         data : body,
         success: function (text) {
             var pretty = JSON.stringify(JSON.parse(text),null,2);
             $(outputDiv).text(pretty);
         },
-        error: function (err) { standardErrorCallback(err, errorDiv) }
+        error: function (err, status, header) {
+            if(err.getResponseHeader("REQUIRES_AUTH") == 1) {
```

Comment:
this code is repeated below, probably better to refactor it to a method like `standardErrorCallback` is and reuse that ?